services/frontend-v3/src/routes/Secured.js

```diff
@@ -193,23 +192,22 @@ function Secured(props) {
                 />
               )}
             />
-            {/****** Logout authorized user ******/}
+            {/** **** Logout authorized user ***** */}
             <Route
               exact
               path="/secured/logout"
               render={(routeProps) => (
                 <Logout keycloak={keycloak} {...routeProps} />
               )}
             />
-            {/****** 404 Page ******/}
+            {/** **** 404 Page ***** */}
```

Comment:
this looks weird

Reply:
Its using the multi-line comment syntax. Since its following the airbnb style standard.
/**
comment 
*/

Reply:
yea i agree with  {/* comment */} or just make it multi-line